src/views/catalog/CreateVMHorizontalNav/CreateVMHorizontalNav.tsx

```diff
@@ -21,18 +21,18 @@ const CreateVMHorizontalNav: FC<RouteComponentProps<{ ns: string }>> = ({
   const { t } = useKubevirtTranslation();
 
   const [currentTab, setCurrentTab] = useState<CREATE_VM_TAB>(
-    location.pathname.endsWith(CREATE_VM_TAB.INSTANCE_TYPES)
-      ? CREATE_VM_TAB.INSTANCE_TYPES
-      : CREATE_VM_TAB.CATALOG,
+    location.pathname.endsWith(CREATE_VM_TAB.TEMPLATE)
+      ? CREATE_VM_TAB.TEMPLATE
+      : CREATE_VM_TAB.INSTANCE_TYPES,
   );
 
   const catalogURL = useMemo(
-    () => `/k8s/${match?.params?.ns ? `ns/${match?.params?.ns}` : ALL_NAMESPACES}/templatescatalog`,
+    () => `/k8s/${match?.params?.ns ? `ns/${match?.params?.ns}` : ALL_NAMESPACES}/catalog`,
     [match],
   );
 
   const handleTabClick = (
-    event: MouseEvent | React.KeyboardEvent | React.MouseEvent<any>,
+    _event: MouseEvent | React.KeyboardEvent | React.MouseEvent<any>,
```

Comment:
This would have been a good opportunity to import these by name and remove the 'React.'

Reply:
sure, updated